test/actions/katello/upstream_subscriptions/remove_entitlement_test.rb

```diff
@@ -15,4 +15,9 @@
     ::Katello::Resources::Candlepin::UpstreamConsumer.expects(:remove_entitlement).with('foo')
     run_action @planned_action
   end
+
+  it 'outputs a message if entitlement is gone' do
+    ::Katello::Resources::Candlepin::UpstreamConsumer.expects(:remove_entitlement).with('foo').raises(::Katello::Errors::UpstreamEntitlementGone)
+    assert_equal(" has already been deleted", run_action(@planned_action).output[:response])
```

Comment:
I'd like to see this test with some given input like this:

```
+++ b/test/actions/katello/upstream_subscriptions/remove_entitlement_test.rb
@@ -8,7 +8,7 @@ describe ::Actions::Katello::UpstreamSubscriptions::RemoveEntitlement do
   before :all do
     @org = FactoryBot.create(:katello_organization)
     set_organization(@org)
-    @planned_action = create_and_plan_action(subject, org_id: @org.id, entitlement_id: 'foo')
+    @planned_action = create_and_plan_action(subject, org_id: @org.id, entitlement_id: 'foo', sub_name: 'Awesome Sub')
   end
 
   it 'runs' do
@@ -18,6 +18,6 @@ describe ::Actions::Katello::UpstreamSubscriptions::RemoveEntitlement do
 
   it 'outputs a message if entitlement is gone' do
     ::Katello::Resources::Candlepin::UpstreamConsumer.expects(:remove_entitlement).with('foo').raises(::Katello::Errors::UpstreamEntitlementGone)
-    assert_equal(" has already been deleted", run_action(@planned_action).output[:response])
+    assert_equal("Awesome Sub has already been deleted", run_action(@planned_action).output[:response])
   end
 end
```

Reply:
Gotcha, fixed it